src/containers/WelcomePage/components/worklist/WorkListTabContent.tsx

```diff
@@ -87,6 +88,16 @@ const WorkListTabContent = ({
 }: Props) => {
   const { t } = useTranslation();
 
+  const searchQuery = useSearch(
+    {
+      "responsible-ids": ndlaId,
+      "page-size": 0,
+      fallback: true,
+      "aggregate-paths": "contexts.rootId",
+    },
+    { enabled: !!ndlaId },
+  );
```

Comment:
Trenger denne for at fag-select ikke skal filtreres basert på valg man gjør i select

Reply:
Hvis det er forvirrende hvorfor man trenger den burde du legge igjen en kommentar i koden!